From 1069f56d6225fbd16cea99d5e9988163465f207b Mon Sep 17 00:00:00 2001
From: Janne Grunau <j@jannau.net>
Date: Thu, 25 Dec 2025 18:49:55 +0100
Subject: [PATCH] HACK: arm64: dts: apple: t8112: Mark ps_atc1_common as always
 on

Works around missing suspend/resume handling in ATC phy for DP-altmode.

Signed-off-by: Janne Grunau <j@jannau.net>
---
 arch/arm64/boot/dts/apple/t8112-jxxx.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/apple/t8112-jxxx.dtsi b/arch/arm64/boot/dts/apple/t8112-jxxx.dtsi
index a2631d28737124..7976360ebf1281 100644
--- a/arch/arm64/boot/dts/apple/t8112-jxxx.dtsi
+++ b/arch/arm64/boot/dts/apple/t8112-jxxx.dtsi
@@ -166,6 +166,10 @@
 &dpaudio1 {
 	status = "okay";
 };
+
+&ps_atc1_common {
+	apple,always-on; /* Needs to stay on for DP-alt suspend/resume */
+};
 #endif
 
 /* USB controllers */
